### ISSUE TITLE
Warning in cli during dev

### ISSUE THREAD

**Issue by universse** [gatsbyjs/gatsby#15750]
Seems like there is an issue with the newest update

```
 warning  in ./.cache/gatsby-browser-entry.js

"export 'enqueue' (reexported as 'prefetchPathname') was not found in './loader'
```

**Comment by peteluffman** [comment]
Yep same here

**Comment by klgh** [comment]
I'm also getting this error

**Comment by billyjacoby** [comment]
Im getting this issue as well

**Comment by cheuk3** [comment]
There is no enqueue export in loader.js from #13985

**Comment by MunifTanjim** [comment]
Related PR: #15735

**Comment by kimbaudi** [comment]
same here

**Comment by wardpeet** [comment]
Fixed in gatsby version 2.13.24
